api/services/prompt_service.py

```diff
@@ -96,15 +96,9 @@ def add_prompt_response(self, ivr_prompt_details, data):
             print("Exception occurred")
 
     def fetch_prompt_response(self, prompt):
-        split_prompt_by_hyphen = helpers.split_prompt_by_hyphen(prompt)
-        split_prompt_by_underscore = helpers.split_prompt_by_underscore(
-            split_prompt_by_hyphen[-1]
-        )
```

Comment:
This function is only used here for entering the data in the registration table.
Checked the earlier values in the database which were generated via this. All of them are single word response so, they will not be impacted by this change.